gocourse02/main.go

```diff
@@ -0,0 +1,136 @@
+package main
+
+import "fmt"
+
+// Struct that represents an Animal
+type Animal struct {
+	name      string // Name of the animal
+	species   string // Species of the animal
+	inTheCage bool   // Flag indicating whether the animal is in a cage
+}
+
+// Struct that represents a Zoo, containing multiple animals
+type Zoo struct {
+	Animals []Animal // Slice of animals in the zoo
```

Comment:
:memo: Ми slices не проходили, не обов'язково було робити.

Щодо коменту. Оскільки `Animals` - експортоване поле, то має бути так (вище лишив лінк на "Go Doc Comments"):

```suggestion
	Animals []Animal // Animals are animals in the zoo.
```

Або так:

```suggestion
	// Animals are animals in the zoo.
	Animals []Animal
```

Reply:
Я взагалі повидаляю ці коменти )